diff --git a/node_modules/@rainbow-me/rainbowkit/dist/chunk-3HHDKJZM.js b/node_modules/@rainbow-me/rainbowkit/dist/chunk-3HHDKJZM.js
index 09bb896..0e8bbf7 100644
--- a/node_modules/@rainbow-me/rainbowkit/dist/chunk-3HHDKJZM.js
+++ b/node_modules/@rainbow-me/rainbowkit/dist/chunk-3HHDKJZM.js
@@ -1107,7 +1107,7 @@ var CloseButton = ({
 function formatAddress(address) {
   const leadingChars = 4;
   const trailingChars = 4;
-  return address.length < leadingChars + trailingChars ? address : `${address.substring(0, leadingChars)}\u2026${address.substring(address.length - trailingChars)}`;
+  return address.length < leadingChars + trailingChars ? address : `${address.substring(0, leadingChars)}...${address.substring(address.length - trailingChars)}`;
 }
 
 // src/components/ConnectButton/formatENS.ts
@@ -1538,7 +1538,7 @@ function ProfileDetails({
   }, /* @__PURE__ */ React26.createElement(Box, {
     textAlign: "center"
   }, /* @__PURE__ */ React26.createElement(Text, {
-    as: "h1",
+    as: "h2",
     color: "modalText",
     id: titleId,
     size: mobile ? "20" : "18",
@@ -1546,7 +1546,7 @@ function ProfileDetails({
   }, accountName)), balanceData && /* @__PURE__ */ React26.createElement(Box, {
     textAlign: "center"
   }, /* @__PURE__ */ React26.createElement(Text, {
-    as: "h1",
+    as: "h2",
     color: "modalTextSecondary",
     id: titleId,
     size: mobile ? "16" : "14",
@@ -1820,8 +1820,10 @@ function ChainModal({
   }, /* @__PURE__ */ React29.createElement(Text, {
     color: "modalText",
     size: "14",
-    weight: "medium"
-  }, "Your wallet does not support switching networks from", " ", appName != null ? appName : "this app", ". Try switching networks from within your wallet instead."))))));
+    weight: "medium",
+    font: "SF Mono",
+    fontFamily: "SF Mono"
+  }, "Your wallet does not support switching networks from this app. Try switching networks from within your wallet instead."))))));
 }
 
 // src/components/ConnectModal/ConnectModal.tsx
@@ -2591,11 +2593,11 @@ function ConnectDetail({
     readyMsg = `${name} is not available on this device`;
   }
   const secondaryAction = showWalletConnectModal ? {
-    description: "Need the official WalletConnect modal?",
+    description: "Open WalletConnect modal →",
     label: "OPEN",
     onClick: showWalletConnectModal
   } : qrCode ? {
-    description: `Don\u2019t have the ${name} app?`,
+    description: `Get the ${name} app →`,
     label: "GET",
     onClick: () => changeWalletStep("DOWNLOAD" /* Download */)
   } : {
@@ -3563,7 +3565,8 @@ function ConnectButton({
       alignItems: "center",
       display: "flex",
       height: "24",
-      paddingX: "4"
+      paddingX: "4",
+      fontFamily: "body"
     }, "Wrong network") : /* @__PURE__ */ React47.createElement(Box, {
       alignItems: "center",
       display: "flex",
@@ -3651,6 +3654,7 @@ function ConnectButton({
       paddingX: "14",
       paddingY: "10",
       transition: "default",
+      fontSize: 16,
       type: "button"
     }, label));
   });
diff --git a/node_modules/@rainbow-me/rainbowkit/dist/components/index.css b/node_modules/@rainbow-me/rainbowkit/dist/components/index.css
index b8a0f96..7ea3103 100644
--- a/node_modules/@rainbow-me/rainbowkit/dist/components/index.css
+++ b/node_modules/@rainbow-me/rainbowkit/dist/components/index.css
@@ -82,6 +82,10 @@
 }
 [data-rk] .ju367v8 {
   display: block;
+  font-family: 'SF Mono';
+  font-weight: 600;
+  letter-spacing: -0.0px;
+  font-size: 18px;
 }
 [data-rk] .ju367va {
   display: flex;
@@ -926,6 +930,10 @@
 }
 [data-rk] .ju367v80 {
   padding-top: 8px;
+  font-family: 'SF Mono';
+  font-weight: 600;
+  letter-spacing: -0.0px;
+  font-size: 18px;
 }
 [data-rk] .ju367v81 {
   padding-top: 10px;
diff --git a/node_modules/@rainbow-me/rainbowkit/dist/index.css b/node_modules/@rainbow-me/rainbowkit/dist/index.css
index b8a0f96..7ea3103 100644
--- a/node_modules/@rainbow-me/rainbowkit/dist/index.css
+++ b/node_modules/@rainbow-me/rainbowkit/dist/index.css
@@ -82,6 +82,10 @@
 }
 [data-rk] .ju367v8 {
   display: block;
+  font-family: 'SF Mono';
+  font-weight: 600;
+  letter-spacing: -0.0px;
+  font-size: 18px;
 }
 [data-rk] .ju367va {
   display: flex;
@@ -926,6 +930,10 @@
 }
 [data-rk] .ju367v80 {
   padding-top: 8px;
+  font-family: 'SF Mono';
+  font-weight: 600;
+  letter-spacing: -0.0px;
+  font-size: 18px;
 }
 [data-rk] .ju367v81 {
   padding-top: 10px;
